src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -0,0 +1,176 @@
+.event-list-item {
+  @include show-svg-on-hover();
+  text-decoration: none;
+  display: grid;
+  padding: $s-lg;
+  background-color: #fcfcfc;
+  margin: 0 auto;
+  grid-template-areas:
+    "image"
+    "content";
+  grid-template-rows: auto;
+  row-gap: $s-md;
+  max-width: $layout__max-width;
+
+  > svg {
+    display: none;
+  }
+}
+
+.event-list-item__image-container {
+  @extend %identity-placeholder-pseudo;
+  grid-area: image;
+  position: relative;
+  width: 100%;
+  aspect-ratio: 4 / 3;
+}
+
+.event-list-item__image {
+  position: absolute;
+  z-index: $z-10;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.event-list-item__content {
+  @include typography($typo__body-medium);
+  display: grid;
+  grid-template-areas:
+    "tag date"
+    "title title"
+    "description description"
+    "location schedule";
+  row-gap: $s-sm;
+}
+
+.event-list-item__tag {
+  grid-area: tag;
+  @include typography($typo__small-caption);
+  width: max-content;
+}
+
+.event-list-item__date {
+  grid-area: date;
+  @include typography($typo__small-caption);
+  text-align: right;
+}
+
+.event-list-item__title {
+  grid-area: title;
+  @include typography($typo__h4);
+  @extend %text-truncate;
+  margin-bottom: $s-sm;
+}
+
+.event-list-item__description {
+  grid-area: description;
+  @include typography($typo__small-caption);
+  @extend %text-ellipsis-two-lines;
+  -webkit-line-clamp: 3;
```

Comment:
Consider if we can make `text-ellipsis-two-lines` into a mixin with a parameter.

It seems as if we would like to add a configurable clamp here.

Reply:
Added a mixin for this with a default of 2, and the ability to change lines as parameter. Refactored existing usages and removed the placeholder.